### PR TITLE
Testnet reset DO NOT MERGE

### LINKED ISSUE #609: Update wallet code for testnet2

### PR DESCRIPTION
**Pull request by jcvernaleo**
Closes #609

### CODE REVIEW

File: walletsetup.go [hunk 1]
```diff
@@ -34,8 +34,8 @@ func networkDir(dataDir string, chainParams *chaincfg.Params) string {
 	// paramaters will likely be switched to being named "testnet" in the
 	// future.  This is done to future proof that change, and an upgrade
 	// plan to move the testnet data directory can be worked out later.
-	if chainParams.Net == wire.TestNet {
-		netname = "testnet"
+	if chainParams.Net == wire.TestNet2 {
+		netname = "testnet2"
```
Comment (jrick) [inline walletsetup.go]: actually I would probably make this a switch:

```
switch chainParams.Net {
case wire.TestNet:
    netname = "testnet"
case wire.TestNet2:
    netname = "testnet2"
}
```

**Comment by jcvernaleo** [comment]
I agree on the warning about the old testnet dir but would rather it be a seperate PR.

File: walletsetup.go (hunk 1 above)
Comment (jcvernaleo) [inline walletsetup.go]: Done.

**Comment by jcvernaleo** [comment]
This will also need a glide update for the updated dcrd.

**Comment by jrick** [comment]
Yep I know, went to go test it and then realized it won't build.

**Pull request merged**